Dockerfile.autotune

```diff
@@ -25,7 +25,7 @@ RUN microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install -y jav
      && microdnf clean all 
 
 RUN mkdir -p /usr/share/maven /usr/share/maven/ref \
-  && curl -fsSL -o /tmp/apache-maven.tar.gz https://dlcdn.apache.org/maven/maven-3/3.9.11/binaries/apache-maven-3.9.11-bin.tar.gz \
+  && curl -fsSL -o /tmp/apache-maven.tar.gz https://archive.apache.org/dist/maven/maven-3/3.9.8/binaries/apache-maven-3.9.8-bin.tar.gz \
```

Comment:
I see 3.9.11 too, shouldn't we use that?

Reply:
@khansaad I suggested using archive so that we dont have links broken every few weeks, however we should use the latest one available now. So lets use 3.9.11 as suggested by Chandrakala

Reply:
Apologies, missed the updated version. 
Have added it now.